common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -350,6 +349,19 @@ object DotcomRenderingDataModel {
       )
     }
 
+    val modifiedFormat = {
+      val ogFormat = content.metadata.format.getOrElse(ContentFormat.defaultContentFormat)
```

Comment:
```suggestion
      val originalFormat = content.metadata.format.getOrElse(ContentFormat.defaultContentFormat)
```
As much as I love the use of "OG" here, brevity could be confusing if anyone comes to look at this in the future and is unsure what `og` stands for.

Reply:
'original gangsta'